session/session.go

```diff
@@ -255,7 +258,7 @@ func (s *session) OpenCh(
 		return perun.ChannelInfo{}, perun.GetAPIError(err)
 	}
 
-	ch := newChannel(pch, openingBals.Currency, parts, s.timeoutCfg, challengeDurSecs)
+	ch := newChannel(pch, openingBals.Currency, []string{perun.OwnAlias, peer.Alias}, s.timeoutCfg, challengeDurSecs)
```

Comment:
Can you use like before, declaring the `parts` and `partsAddress` variables next to each other, so that it is visually easier to ensure that the order or participants is same on both the arrays ?

Reply:
Updated to old structure of declaring `parts`  and `partsAddress` variable in this [commit.](https://github.com/hyperledger-labs/perun-node/pull/110/commits/419c09f1a62eb87b03be5594f43da29dfd7ed31c)